command/src/worker.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
Weird change

Reply:
I forgot to delete the file, sorry. All the content was dispatched in `request.rs` and `response.rs`, for clarity.